english/dig/dig.md

```diff
@@ -8,7 +8,7 @@ It says "hello"
 
 "Hello, Mr Bird!", you say, and spot a collar with a tag on it.
 
-Upon inspecting the tag, you see it reads: "Nevermore". Turning over the tag, you see: "If found, please return to:
+Upon inspecting the tag, you see it reads: "Nevermore: Rub me three times". Turning over the tag, you see: "If found, please return to:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.